postgres/conf.yaml.example

```diff
@@ -68,3 +68,8 @@ instances:
 #    Collect database size metrics. Default value is True but they might be slow with large databases
 #    collect_database_size_metrics: False
 #
+
+#    Include statistics from the default database 'postgres' in the check metrics, suggested value
+#    is false
+#    do_collect_default_database: False
```

Comment:
Could you rename the option: `collect_default_database`

Reply:
👍